sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/servicebus/AzureServiceBusOperationAutoConfiguration.java

```diff
@@ -31,9 +31,9 @@
 @ConditionalOnClass(ServiceBusReceivedMessage.class)
 @ConditionalOnProperty(value = "spring.cloud.azure.servicebus.enabled", matchIfMissing = true)
 @EnableConfigurationProperties(AzureServiceBusProperties.class)
-public class AzureServiceBusAutoConfiguration {
+public class AzureServiceBusOperationAutoConfiguration {
```

Comment:
what does operation mean?
and I think we should discuss this resorucemanager support, how we deal with that?